### PR TITLE
Add Backup and restore with Velero and Restic for podified

### PR DESCRIPTION
**Pull request by dehawkins512**
@bdunne  FYI

### CODE REVIEW

**Comment by dehawkins512** [comment]
@miq-bot assign @Fryguy

**Comment by Fryguy** [comment]
@bdunne Please review

File: general_configuration/_topics/configuration.md [hunk 1]
```diff
@@ -3080,9 +3080,33 @@ from the global copy.
...
 
-#### Backing Up and Restoring a Database
-
-##### Running a Single Database Backup
+#### Backing Up and Restoring {{ site.data.product.title }} in a Kubernetes Based Environment (podified)
+
+**Prerequisites:**
+   - Velero and Restic must be installed and working in your cluster. For more information on Velero, see the [Velero documentation](https://velero.io/docs/v1.5/). For more information on Restic, see the [Restic documentation](https://restic.readthedocs.io/en/stable/) 
+
+**Note:** The operator adds labels and/or annotations to everything that needs to be backed up. You can easily and quickly backup only what is required. The operator will add annotations for restic volume backups for example, `backup.velero.io/backup-volumes: miq-pgdb-volume`.
```
Comment (bdunne) [inline general_configuration/_topics/configuration.md:3088]: Should we make the example more generic or leave it out to simplify productization of the documentation?

File: general_configuration/_topics/configuration.md [hunk 2]
```diff
@@ -3080,9 +3080,33 @@ from the global copy.
...
+
+1. The backup label key can be configured by adding the following to the CR. The default applies the label `manageiq.org/backup=t` if nothing else is specified.
```
Comment (bdunne) [inline general_configuration/_topics/configuration.md:3090]: Same here, but we probably want to make manageiq.org/backup=t a config parameter

File: general_configuration/_topics/configuration.md [hunk 3]
```diff
@@ -3080,9 +3080,33 @@ from the global copy.
...
+   ```
+   ...
+   spec:
+     ...
+     backupLabelName: <name of your openshift OIDC CA cert> (default: manageiq.org/backup)
```
Comment (bdunne) [inline general_configuration/_topics/configuration.md:3095]: Paste error? `<name of your openshift OIDC CA cert>`

File: general_configuration/_topics/configuration.md [hunk 4]
```diff
@@ -3080,9 +3080,33 @@ from the global copy.
...
+   ```
+2. Create a backup by running the command:
+   ```
+   velero backup create <your backup name> --include-namespaces <your namespace> -l manageiq.org/backup=t
```
Comment (bdunne) [inline general_configuration/_topics/configuration.md:3099]: Same comment about the config parameter here

File: general_configuration/_topics/configuration.md (hunk 3 above)
Comment (bdunne) [reply general_configuration/_topics/configuration.md:3095]: Hmm... looks like the copy/paste error was on my side in the pods repo.

Comment (bdunne) [inline general_configuration/_topics/configuration.md:3095]:
```suggestion
     backupLabelName: <key for the label that will be used to identify resources to be backed up> (default: manageiq.org/backup)
```

File: general_configuration/_topics/configuration.md (hunk 2 above)
Comment (Fryguy) [reply general_configuration/_topics/configuration.md:3090]: @bdunne when you say a config parameter you mean put it in https://github.com/ManageIQ/manageiq-documentation/blob/master/_data/product.yml ?

File: general_configuration/_topics/configuration.md (hunk 1 above)
Comment (Fryguy) [reply general_configuration/_topics/configuration.md:3088]: @bdunne Can we say `backup.velero.io/backup-volumes: <volume-label>` ?

**Review by bdunne — APPROVED** [review]

**Comment by Fryguy** [comment]
Backport details:

```
commit 8bd1166e222f72e7b79386529584c6bd8d988ed2 (HEAD -> kasparov, upstream/kasparov)
Author: Brandon Dunne <bdunne@redhat.com>
Date:   Thu Feb 4 18:11:15 2021 -0500

    Merge pull request #1526 from dehawkins512/dan_updates_for_bkup_restore

    Add Backup and restore with Velero and Restic for podified

    (cherry picked from commit 3dc251f32f35bf9ffb800dd744ba3989d94137c5)
```

**Pull request merged**